tests/functional/conftest.py

```diff
@@ -104,8 +104,6 @@ def pytest_generate_tests(metafunc):
                     for url in lang_urls:
                         doc = get_web_page(f"{base_url}{url}")
                         download_urls = [a.attrib["href"] for a in doc("a.download-link")]
-                        # Only check links to download.mozilla.org
-                        download_urls = [link for link in download_urls if link.startswith("https://download.mozilla.org/")]
                         for url in download_urls:
```

Comment:
The tests failed on dev but because dev uses a bouncer URL like `https://bouncer-bouncer.stage.mozaws.net/?product=firefox-latest-ssl&os=win64&lang=en-US`.

But I don't think this check is necessary since we're already filtering the links based on `a.download-link` elements.

Reply:
I think it might be available as an env var `BOUNCER_URL`